mitosheet/mitosheet/mito_config.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# Copyright (c) Saga Inc.
+# Distributed under the terms of the GPL License.
+
+from typing import Any, Dict, Optional
+
+# These variables must match the variables defined in 
+# mito_config.py of the mitosheet_helper_config package
+MEC_VERSION = 'mec_version'
+SUPPORT_EMAIL = 'support_email'
```

Comment:
Can we prefix both of these to `MEC_CONFIG_KEY_VERSION`, and `MEC_CONFIG_KEY_SUPPORT_EMAIL`, so that it's clear they are keys?